Library/Homebrew/livecheck.rb

```diff
@@ -1,19 +1,19 @@
 # frozen_string_literal: true
 
-# The `Livecheck` class implements the DSL methods used in a formula's
+# The `Livecheck` class implements the DSL methods used in a formula/cask's
 # `livecheck` block and stores related instance variables. Most of these methods
 # also return the related instance variable when no argument is provided.
 #
 # This information is used by the `brew livecheck` command to control its
 # behavior.
 class Livecheck
-  # A very brief description of why the formula is skipped (e.g., `No longer
+  # A very brief description of why the formula/cask is skipped (e.g., `No longer
   # developed or maintained`).
   # @return [String, nil]
   attr_reader :skip_msg
 
-  def initialize(formula)
-    @formula = formula
+  def initialize(formula_or_cask)
```

Comment:
We should initialize `@version` to `nil`, like the others here. [It would go at the end, since these are in alphabetical order.]